src/test/resources/playSong_Smoke.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="TestNG">
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="Homework18"/>
+             <methods>
+                 <include name="playSong"/>
+             </methods>
+        </classes>
+    </test>
+</suite>
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/212079812-8fe55060-6087-4afa-a0d2-959e081ca38a.png)